homeassistant/components/twinkly/light.py

```diff
@@ -249,6 +253,9 @@ async def async_turn_on(self, **kwargs: Any) -> None:
                 await self._client.set_current_movie(int(movie_id))
                 await self._client.set_mode("movie")
                 self._client.default_mode = "movie"
+        else:
+            await self._client.set_brightness(self._attr_internal_brightness)
```

Comment:
Should we also set a `_client.default_mode` here to ensure that the right mode is set when `_client.turn_on()` is run later?